### PR TITLE
Vendored in latest changes from px-backup-api for 2.7.0

### PR DESCRIPTION
**Pull request by mkoppal-px**
**What this PR does / why we need it**:
Vendored in latest changes from px-backup-api for 2.7.0

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: tests/common.go [hunk 1]
```diff
@@ -10,48 +10,22 @@ import (
...
-	"net/http"
-	"regexp"
-	"runtime"
 
 	"cloud.google.com/go/storage"
-	appType "github.com/portworx/torpedo/drivers/applications/apptypes"
-	"google.golang.org/api/iterator"
-	"google.golang.org/api/option"
-	"github.com/portworx/torpedo/pkg/stats"
-	optest "github.com/libopenstorage/operator/pkg/util/test"
```
Comment (kshithijiyer-px) [inline tests/common.go:32]: Why are these things changing?

**Review by kshithijiyer-px — CHANGES_REQUESTED** [review]
The PR looks good but don't understand the changes in common.go can you please check

File: tests/common.go (hunk 1 above)
Comment (kphalgun-px) [reply tests/common.go:32]: I think go fmt automatically sorts imports alphabetically. First comes standard library import and then external library imports

Comment (mkoppal-px) [reply tests/common.go:32]: Goland formatted it. No other changes in this file

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Review by dbhatnagar-px — APPROVED** [review]
Lgtm

**Pull request merged**